gfx/models/portraits/attachments/female_headgears/secular/goblin/gallywix_hat_01/female_headgear_secular_goblin_gallywix_hat_01.asset

```diff
@@ -0,0 +1,19 @@
+﻿pdxmesh = {
```

Comment:
The female top hat clips with the head sometimes.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114947423-2f24fe00-9e5e-11eb-96d3-e7b978475f9c.png)

</details>